pkg/jaeger/query/trace_query.go

```diff
@@ -237,8 +240,22 @@ func (b *Builder) buildTraceIDSubquery(q *spanstore.TraceQueryParameters) (strin
 				clauses = append(clauses, qual)
 			default:
 				//TODO make sure this is optimized correctly
-				params = append(params, k, "\""+v+"\"")
-				qual := fmt.Sprintf(`_ps_trace.tag_map_denormalize(s.span_tags)->$%d = $%d`, len(params)-1, len(params))
+				val := "\"" + v + "\""
```

Comment:
Nit:
```
if digitCheck.MatchString(v) 
  val = v
else
 val = "\"" + v + "\""

Reply:
Subjective. The current one is less blocks so will be happy to continue with that.